src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -306,8 +316,8 @@ namespace HeatPumpWaterToWaterSimple {
 		int NumAlphas; // Number of elements in the alpha array
 		int NumNums; // Number of elements in the numeric array
 		int IOStat; // IO Status when calling get input subroutine
-		Array1D_string AlphArray( 5 ); // character string data
-		Array1D< Real64 > NumArray( 15 ); // numeric data
+		Array1D_string AlphArray( 6 ); // character string data
+		Array1D< Real64 > NumArray( 17 ); // numeric data
```

Comment:
Yeah, that can happen, I don't think there are any nested gets here that would trip up.  I can make this change